src/main/java/org/example/homework10/Album/Album.java

```diff
@@ -0,0 +1,47 @@
+package org.example.homework10.Album;
+
+import java.util.ArrayList;
+
+public class Album {
+
+    class SongList {
+
+        ArrayList<Song> songs;
```

Comment:
Not correct implementation. Please read it again and rework
"Two fields: a String called "title" to represent the title of the album, and an instance of the SongList inner class to manage the list of songs. "